sale_order_value_proposal_by_partner_shipping/models/res_partner.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    warehouse_id = fields.Many2one("stock.warehouse", string="Warehouse")
```

Comment:
Just came to think that this field should actually be a property field to make it more generic.  What do you think?